tests/parametric/conftest.py

```diff
@@ -762,7 +761,13 @@ def write_stable_config(self, stable_config: dict, path: str, test_library: APML
         self.write_stable_config_content(stable_config_content, path, test_library)
 
     def write_stable_config_content(self, stable_config_content: str, path: str, test_library: APMLibrary) -> None:
-        cmd = f'bash -c "mkdir -p {Path(path).parent!s} && printf {shlex.quote(stable_config_content)} | tee {path}"'
+        # Base64 encode the YAML content to avoid shell issues
+        encoded = base64.b64encode(stable_config_content.encode()).decode()
```

Comment:
as you have modified the test logic, you must test it across all tracer, could you remove the `[java]` prefix on PR title ?